presto-spi/src/main/java/com/facebook/presto/spi/block/MapBlock.java

```diff
@@ -201,7 +201,7 @@ private MapBlock(
                 + valueBlock.getRetainedSizeInBytes()
                 + sizeOf(offsets)
                 + sizeOf(mapIsNull)
-                + sizeOfIntArray(keyBlock.getPositionCount() * HASH_MULTIPLIER);  // hashtable size if it was built
+                + hashTables.getRetainedSizeInBytes();  // The instance size for HashTables object because it always is not null.
```

Comment:
Is the comment on this line really necessary?

Reply:
Removed